learning/inference.py

```diff
@@ -269,6 +275,31 @@ def main():
     inference_data_table = wandb.Table(columns=table_cols, data=inference_data)
     run.log({"Inference Data": inference_data_table})
 
+    video_name = (str(agent.image_directory).split("/"))[-1]
+    filelist_name = video_name + "_filelist.txt"
+
+    image_files = sorted([f for f in agent.image_directory.iterdir()])
+    with open(filelist_name, "w") as file_out:
+        for file in image_files:
+            file_out.write(f"file '{Path(file)}'\n")
```

Comment:
Use pathlib for this. Check out [Path.glob](https://docs.python.org/3/library/pathlib.html#pathlib.Path.glob)